### ISSUE TITLE
1.0.0-SNAPSHOT doesn't work?

### ISSUE THREAD

**Issue by radicaled** [TomasMikula/RichTextFX#302]
I've just upgraded to 1.0.0-SNAPSHOT and see this message:

```
Exception in thread "main" java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$99(LauncherImpl.java:182)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NoSuchMethodError: org.reactfx.value.Val.create(Ljava/util/function/Supplier;Lorg/reactfx/EventStream;)Lorg/reactfx/value/Val;
```

This is what `gradlew dependencies` gives for richtextfx:

```
+--- org.fxmisc.richtext:richtextfx:1.0.0-SNAPSHOT
|    +--- org.reactfx:reactfx:2.0-SNAPSHOT -> 2.0-M4u1
|    +--- org.fxmisc.wellbehaved:wellbehavedfx:0.3
|    +--- org.fxmisc.flowless:flowless:0.5-SNAPSHOT
|    |    \--- org.reactfx:reactfx:2.0-M4u1
|    \--- org.fxmisc.undo:undofx:1.2
|         \--- org.reactfx:reactfx:2.0-M4u1
```

**Comment by TomasMikula** [comment]
For some reason "your" gradle seems to override `reactfx:2.0-SNAPSHOT` by `reactfx:2.0-M4u1`, which is pulled in by the other dependencies. "My" gradle uses `reactfx:2.0-SNAPSHOT`.

**Comment by TomasMikula** [comment]
It might be time to make a new ReactFX release anyway.

**Comment by TomasMikula** [comment]
I published a new snapshot depending on ReactFX 2.0-M5. Let's see if that helps.

**Comment by radicaled** [comment]
Success!

I had to specify 2.0-M5 explicitly, though, by adding it as a top-level dependency. I tried to do that with  2.0-SNAPSHOT earlier and it didn't work. Not sure what the deal is.
